packages/design/src/components/UpsertModal.re

```diff
@@ -29,6 +29,7 @@ let make = (~method, ~displayName, ~opener, ~content, ~className="") =>
                switch (method) {
                | `insert => "Insert"
                | `update => "Update"
+               | `remove => "Remove"
```

Comment:
eita this is not exhaustive?
any more logs in the compiler?

Reply:
Yes, I don't know why we have it here. We can just have the displayName